docs/examples/acme-issuer.yaml

```diff
@@ -27,8 +27,21 @@ spec:
           project: gcloud-prod-project
       - name: cf-dns
         cloudflare:
+          # The cloudflare user account email
           email: user@example.com
           # A secretKeyRef to a cloudflare api key
           apiKeySecretRef:
             name: cloudflare-api-key
             key: api-key.txt
+      - name: route53
+        route53:
+          # The Route53 access key ID
+          accessKeyID: AKIADKOU3GLWAQM8WQKJ
+          # A secretKeyRef to a the route53 secret access key
+          secretAccessKeySecretRef:
+            name: route53-config
+            key: secret-access-key
+          # The Route53 hosted zone ID
+          hostedZoneID: DIKER8JPL21PSA
```

Comment:
This field is optional, and only required if cert-manager cannot disambiguate between two different hosted zones for the same zone name. It may be worth noting that here?

Some more info here: https://github.com/jetstack-experimental/cert-manager/issues/94